examples/pydax-mvp-demo.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
```

Comment:
<p>I would insert a cell above this cell that runs:</p><p> </p><pre class="ql-syntax" spellcheck="false">Load a dataset is simple: One single line you'll have a remote dataset readily loaded, no hassle!
</pre><p> </p><pre class="ql-syntax" spellcheck="false">dataset = pydax.load_dataset('wikitext103')  # Load a dataset
dataset['valid'][:730]  # preview my dataset
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/pydax/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Actually if possible this can be put at the top. The most important thing is to get people to be interested, then they can learn a bit expanded basic usage/highlighted features. After that, they would be able to read the document and figure out advanced features.</p>